.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
   - DJANGO_VERSION=1.5
```

Comment:
Good point.  The same is true for 1.6 at the moment, and will be true of 1.4 soon.